model/database/registry.yaml

```
@@ -541,9 +542,50 @@ groups:
               value: "ConsistentPrefix"
               stability: experimental
         stability: experimental
-        brief: Account or request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
+        brief: >
+          Effective Request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
+          This represents the first applicable consistency level among those specified by the request or the client.
         examples: ["Eventual", "ConsistentPrefix", "BoundedStaleness", "Strong", "Session"]
 
+  - id: registry.db.cosmosdb.network
+    type: attribute_group
+    display_name: Azure Cosmos DB Network level Attributes
+    stability: experimental
+    brief: >
+      This group defines attributes for Azure Cosmos DB Network call.
+    attributes:
+      - id: db.cosmosdb.network.host
+        type: string
+        stability: experimental
+        brief: Specifies the host address from the Gateway or Backend Service endpoint.
+        examples: ["rntbd://cdb-east-us.azure.document.com"]
+      - id: db.cosmosdb.network.port
+        type: int
+        stability: experimental
+        brief: The port used for the Gateway or Backend Service endpoint.
+        examples: 443
+      - id: db.cosmosdb.network.response.status_code
+        type: int
+        stability: experimental
+        brief: The HTTP status code returned from the network request.
```

I think HTTP is misleading here. If I understand correctly, the protocol happens to reuse HTTP status codes, but it's not necessarily HTTP, right?

Consider something like:

```suggestion
        brief: The status code returned in response.
        note: The protocol reuses HTTP status codes.
```